File: Examples/05_Library/Library/ViewController.m
//
//  ViewController.m
//  SimpleApp
//
//  Created by Tom Yu on 2020/10/12.
//

#import "ViewController.h"
#import <LibraryPod/Cat.h>
#import <LibraryPod/Dog.h>
#import <LibraryPod/Rose.h>
#import <LibraryPod/AppleTree.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[[Cat alloc] init] run];
    [[[Dog alloc] init] eat];
    [[[Rose alloc] init] bloom];
    [[[AppleTree alloc] init] fruit];
}

@end

File: Examples/01_Simple/SimpleSwift/ViewController.swift
//
//  ViewController.swift
//  SimpleSwift
//
//  Created by Tom Yu on 2021/5/31.
//

import UIKit


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        Cat().run();
        Dog().eat();
        Rose().bloom();
        AppleTree().fruit();
    }
}

